_includes/integrations/angular-tech-ref.md

```diff
@@ -275,6 +275,51 @@ For information setting the toolbar for {{site.productname}}, see: [User interfa
 ></editor>
 ```
 
+#### `modelEvents` (since v4.0.0)
```

Comment:
I don't know if integrations have this, but for core editor docs there's a `{{ site.requires_... }}` which will expand into a nice message saying "this feature is only available in this situation". Is there one for integrations, and should this be using it?

Reply:
fixed in https://github.com/tinymce/tinymce-docs/pull/1681/commits/cd246890b556e0460ba5cdde28eece9d264ffc55